ui/src/components/Socket/SocketDetails.vue

```diff
@@ -91,6 +91,12 @@
             <td></td>
           </tr>
 
+          <tr>
+            <td class="key-column">{{ $t("data") }}</td>
+            <td>{{ JSON.stringify(socket.data) }}</td>
```

Comment:
Would it make sense to use something like this:

```html
<pre><code>{{ JSON.stringify(socket.data, null, 2) }}</code></pre>
```

to properly format the data?